app/src/main/kotlin/com/wire/android/ui/home/conversationslist/ConversationRouter.kt

```diff
@@ -53,13 +55,18 @@ fun ConversationRouterHomeBridge(
         ConversationState()
     }
 
+    val conversationOptionSheetState = remember(conversationState.conversationSheetContent) {
```

Comment:
I don't see a problem with observing this other state, but, maybe @trOnk12  can take a look, since he recently refactored some of this code 👨‍🔬  

I can imagine that passing directly `conversationState.conversationSheetContent` wasn't an option ?

Reply:
I also do not see a problem I think, it gives us a option to control the state of it outside the parent so I think it is even better